.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+<!--- Provide a general summary of your changes in the Title above -->
+<!--- Pull request titles must use the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) format -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+
+## Motivation and Context
```

Comment:
Shouldn’t it go first?

Reply:
I think it does not matter much which is first. I always want to know "what" (the description). The "why" (motivation/context) may be obvious from the description and not need context (or I may not care about it sometimes 🙂). But basically, the template generator I used put them this way and I thought it made sense 😄

Reply:
I just prefer to follow the Agile product-first approach where you describe a problem first and then - solution. In DIPs we do the same ("Motivation"). Also, you see a reference to the issue first where you can get info about that. But you right, it's not so important so it's up to you :).

BTW, is it possible to share these templates among our repos to be able to update them only in one place?

Reply:
Previously we had motivation on the top https://github.com/dashevo/wallet-lib/pull/120

Reply:
I changed the heading order and adjusted the wording like the wallet-lib example.

Reply:
> BTW, is it possible to share these templates among our repos to be able to update them only in one place?

Unfortunately no :slightly_frowning_face: